sui/deploy-contract.js

```diff
@@ -332,9 +332,10 @@ async function upgrade(keypair, client, supportedPackage, policy, config, chain,
 
 async function mainProcessor(args, options, processor) {
     const config = loadConfig(options.env);
-    const [keypair, client] = getWallet(config.sui, options);
-    await printWalletInfo(keypair, client, config.sui, options);
-    await processor(keypair, client, ...args, config, config.sui, options);
+    const sui = getChainConfig(config, 'sui');
```

Comment:
define a chainName base option that defaults to `sui` but can be overriden similar to stellar. We'll deploy `sui-test` name on devnet initially until the final release is ready